lib/crewai/src/crewai/cli/tools/main.py

```diff
@@ -162,9 +162,13 @@ def login(self) -> None:
 
         if login_response.status_code != 200:
             console.print(
-                "Authentication failed. Verify if the currently active organization access to the tool repository, and run 'crewai login' again. ",
+                "Authentication failed. Verify if the currently active organization can access the tool repository, and run 'crewai login' again.",
                 style="bold red",
             )
+            console.print(
+                f"[{login_response.status_code} error - {login_response.json().get('message', 'Unknown error')}]",
```

Comment:
IIRC calling json() may fail if the response body is HTML. It would be good to add a try/except block here to handle that scenario